lib/m2ts/m2ts.js

```diff
@@ -231,20 +267,29 @@ TransportParseStream = function() {
     }
   };
 
+  this.triggerData_ = function(data) {
+    sinceFlushDataEmitted++;
+    totalDataEmitted++;
+    this.trigger('data', data);
+  };
+
+  this.flush = function() {
+    if (sinceFlushDataEmitted === 0) {
+      this.error('flush called before data was emitted in TransportParseStream');
+    }
+    sinceFlushDataEmitted = 0;
+    this.trigger('done');
+  };
+
   this.processPes_ = function (packet, offset, result) {
     result.streamType = this.programMapTable[result.pid];
     result.type = 'pes';
     result.data = packet.subarray(offset);
-
-    this.trigger('data', result);
+    this.triggerData_(result);
   };
 
 };
 TransportParseStream.prototype = new Stream();
-TransportParseStream.STREAM_TYPES  = {
```

Comment:
it was not actually used anywhere. we have all the stream types defined in a separate file called stream-types